apps/browser/qml/pages/components/PrivacySettingsConfirmDialog.qml

```diff
@@ -0,0 +1,149 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+Dialog {
+
+    property alias historyEnabled: historyItem.visible
+    property alias cookieAndSiteDataEnabled: cookieAndSiteDataItem.visible
+    property alias passwordsEnabled: passwordsItem.visible
+    property alias cacheEnabled: cacheItem.visible
+    property alias bookmarksEnabled: bookmarksItem.visible
+    property alias sitePermissionsEnabled: sitePermissionsItem.visible
+
+    property int historyPeriod
+
+    property int _siteDataUsage
+    property int _cacheUsage
+
+    acceptDestinationAction: PageStackAction.Pop
+
+    SilicaFlickable {
+        anchors.fill: parent
+        contentHeight: column.height
+
+        Column {
+            id: column
+
+            width: parent.width
+
+            spacing: Theme.paddingMedium
+
+            DialogHeader {
+                id: header
+
+                //% "Are you sure you want to clear the browser data?"
+                title: qsTrId("sailfish_browser-ti-confirm_clearing_browser_data");
+                //% "Clear"
+                acceptText: qsTrId("sailfish_browser-he-clear");
+            }
+
+            Label {
+                x: Theme.horizontalPageMargin
+                width: parent.width - 2*Theme.horizontalPageMargin
+
+                //% "The following browser data will be deleted:"
+                text: qsTrId("sailfish_browser-la-following_browser_data_will_be_deleted:");
+                highlighted: true
+                wrapMode: Text.Wrap
+            }
+
+            BrowserDataItem {
+                id: historyItem
+
+                text: {
+                    switch(historyPeriod) {
+                    case 1:
+                        //% "Clear browser history for the last 24 hours"
```

Comment:
Too much repetition, it very much clear already that browsing history is cleared